boards/bytesatwork/bytesensi_l/doc/index.rst

```diff
@@ -0,0 +1,224 @@
+| 4     | INT          | Interrupt at P0.13      |
++-------+--------------+-------------------------+
+| 5     | I2C_ADDR     | tied to VBAT            |
++-------+--------------+-------------------------+
+| 6     | GND          | Ground                  |
++-------+--------------+-------------------------+
+
+One Wire Sensor @ X2
+
++-------+----------------+-------------------------+
+| PIN # | Signal Name    | Function                |
++=======+================+=========================+
+| 1     | VDD            | 4V8                     |
++-------+----------------+-------------------------+
+| 2     | IO             | One Wire                |
++-------+----------------+-------------------------+
+| 3     | GND            | Ground                  |
++-------+----------------+-------------------------+
+
+External BLE Antenna @ J1
+
+External LoRa Antenna @ J2
+
+External GPS Antenna @ J3
+
+Programming and Debugging
+*************************
+
+Flashing
+========
+The byteSENSI-L board can be flashed with the SEGGER JLink programmer.
+You can build the following samples as demo application:
+
+#. Build the :ref:`hello_world` sample:
+
+   .. zephyr-app-commands::
+      :zephyr-app: samples/hello_world
+      :board: bytesensi_l
+      :goals: build flash
+      :compact:
+
+#. Build the :ref:`lps22hb` sample:
+
+   .. zephyr-app-commands::
+      :zephyr-app: samples/sensor/lps22hb
+      :board: bytesensi_l
+      :goals: build flash
+      :compact:
+
+#. Build the :ref:`ti_tmp116_sample` sample:
+
+   .. zephyr-app-commands::
+      :zephyr-app: samples/sensor/tmp116
+      :board: bytesensi_l
+      :goals: build flash
+      :compact:
+
+#. Build the :ref:`apds9960` sample:
+
+   .. zephyr-app-commands::
+      :zephyr-app: samples/sensor/apds9960
+      :board: bytesensi_l
+      :goals: build flash
+      :compact:
+
+#. Build the :ref:`ccs811` sample:
+
+   .. zephyr-app-commands::
+      :zephyr-app: samples/sensor/ccs811
+      :board: bytesensi_l
+      :goals: build flash
+      :compact:
+
+#. Build the LoRa samples:
+
+   Using two byteSENSI-L, you can run the LoRa send and receive demo:
+
+   .. zephyr-app-commands::
+      :zephyr-app: samples/drivers/lora/receive
+      :board: bytesensi_l
+      :goals: build flash
+      :compact:
+
+   .. zephyr-app-commands::
+      :zephyr-app: samples/drivers/lora/send
+      :board: bytesensi_l
+      :goals: build flash
+      :compact:
+
+#. Build the jesd216 flash sample:
+
+   .. zephyr-app-commands::
+      :zephyr-app: samples/drivers/jesd216/
+      :board: bytesensi_l
+      :goals: build flash
+      :compact:
+
+#. Build a :ref:`bluetooth-samples`:
+
+   Most of the bluetooth samples should work out of the box, for example,
+   building the :ref:`peripheral_hids` sample:
+
+   .. zephyr-app-commands::
+      :zephyr-app: samples/bluetooth/peripheral_hids
+      :board: bytesensi_l
+      :goals: build flash
+      :compact:
```

Comment:
Please only list one, there is no point in enumerating so many sample applications

Reply:
Alright, I'll keep only the ``hello world`` sample, thanks!